packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -599,7 +599,7 @@ export const createCoinjoinAccount =
         );
 
         // start discovery
-        dispatch(fetchAndUpdateAccount(account.payload));
+        return dispatch(fetchAndUpdateAccount(account.payload));
```

Comment:
This probably changes `isLoading` logic in `AddCoinjoinAccountButton`, but I've tried it and everything looks just fine.

Reply:
Yeah, by coincidence I'm just fixing there right now.

related to https://github.com/trezor/trezor-suite/issues/8380
related to https://github.com/trezor/trezor-suite/pull/8400